README.md

```diff
@@ -202,10 +202,16 @@ In Native mode you can only `stealFocus` to some component to flag it as `focuse
 * **true**
 
 ##### `throttle`: integer
-Enable to throttle the function fired by the event listener.
+Enable to throttle the function fired by the event listener `keydown` and stop it by `keyup`.
 
 * **0 (default)**
 
+##### `throttleKeypresses`: boolean
+In combination with `throttle` it prevents to stop throttling by `keyup`.
```

Comment:
This is not super readable. Can we rephrase it, please?

Keep in mind that this library will be used by other people that would like to understand what is going on through the README and changelog updates.

Either we go for what you indicated above, being very explicit (which also entails the need to phrase ourselves clearly) or we do not.

Also I don't necessarily see any value in having different phrasings in the README as compared to the CHANGELOG, so we could probably save ourselves some time by just using the same texts between the two.

Reply:
CHANGELOG should represent the latest added changes and README should describe features and options, since they have a different purpose, would you like to have them with the same texts?

Reply:
I simplified it at changelog and update docs.

Reply:
All I am saying is (in my opinion) it is sensible to use descriptions of things that are reflective of each other between the two files.